pyproject.toml

```diff
@@ -16,7 +16,7 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
     "Natural Language :: English",
 ]
-dependencies = ["polars==0.20.27", "numpy", "pyarrow", "pandas"]
+dependencies = ["polars>=0.20,<1.13", "numpy", "pyarrow", "pandas"]
```

Comment:
I would prefer if we allow minor version bumps as is common practice for libraries. Did you see breaking changes in the minor versions before or after the 1.0.0 release? I think the software standard is that before the 1.0.0 release breaking changes can occur (e.g. from 0.4.2 to 0.5.0) but after the 1.0.0 release breaking changes can only be released on major versions (i.e. 2.0.0)

Reply:
Looking through the polars release logs I don't see any breaking changes really. some deprecation warnings

Reply:
I will double check again.  It may have just been a deprecation warning.  I looked through the release notes with the team a week and a half before this and thought I remembered something strange in a 1.x minor release.  We're in agreement.  If I don't find anything lets allow minor bumps.

Reply:
Here's one example "Unstable Breaking Changes" this is from Polars 1.5 -> 1.6.  They do have a caveat that "These API's were marked unstable and are allowed to change."

![image](https://github.com/user-attachments/assets/93cb63d6-a49e-4657-a5b5-8f074dd4b183)

Reply:
I'm going to assume that the caveat is an exception and not the rule for their release patterns.  Going to allow minor bumps

Reply:
Yup sounds good to me!